Deploy-CAM.ps1

```diff
@@ -1186,6 +1129,11 @@ function Deploy-CAM()
 	$CAMConfig.internal.rwLocalSecretName = 'remoteWorkstationLocalAdminPassword'
 	$CAMConfig.internal.csLocalSecretName = 'connectionServiceLocalAdminPassword'
 	$CAMConfig.internal.saSasTokenSecretName = "userStorageAccountSaasToken"
+	$CAMConfig.internal.blobUriSecretName = "userStorageAccountUri"
```

Comment:
If this is how we are saving them into key vault, i think we should let primesoft know that the instructions for retrieving this information for the front end should probably change, the examples i gave were using these as separate secrets